openrtb/osversion.go

```diff
@@ -1,17 +1,11 @@
 package openrtb
 
 import (
-	"strconv"
+	"github.com/Vungle/vungo/openrtb/util"
 )
 
-// OsVersion type represents the OS versions of commonly known mobile devices, such as iOS, Android,
-// and Window. The OS version considers only numeric parts to be valid and takes the generic semver
-// like format of MAJOR.MINOR.[BUILD.]PATCH<any-trailing-parts>. The optional build part currently
-// only exists for Windows devices; Custom, unofficial releases containing "alpha", "beta", "rc" are
-// considered insignificant. Therefore, "1.1.2" and "1.1.2rc" are considered the same. Also, each
-// part, separated by the '.' character, is optional; the significance is compared sequentially at
-// its 0th index of the version string. Empty version string, or version string that does not start
-// in numeric characters are considered as version 0.
+// OsVersion is a wrapper type of util.Version for special handling needed exclusively for
```

Comment:
Please don't expose implementation detail, osversion should specified by how os version comparison works, and util.Version will provide the implemention for us but not only one.